testapps/integration/web/index.php

```diff
@@ -29,12 +29,12 @@
 });
 
 $app->post('/logging_standard', function (Request $request) {
-    $logName = $request->get('log_name');
-    $token = $request->get('token');
+    $logName = $request->request->get('log_name');
```

Comment:
I got it. Then it is worth testing logs on stderr. I moved the current test to logging_custom and use stderr in logging_standard. @chingor13 PTAL